installation/scripts/minikube.sh

```diff
@@ -101,6 +102,19 @@ function checkMinikubeVersion() {
     fi
 }
 
+function checkKubectlVersion() {
+    local currentClientVersionMajor=$(kubectl version --client --short | grep -o '[0-9]\+' | sed -n '1p')
+    local currentClientVersionMinor=$(kubectl version --client --short | grep -o '[0-9]\+' | sed -n '2p')
+    local desiredClientVersionMajor=$(echo $KUBECTL_CLI_VERSION | cut -d"." -f1)
+    local desiredClientVersionMinor=$(echo $KUBECTL_CLI_VERSION | cut -d"." -f2)
+    local minorVersionDifference=$(( currentClientVersionMinor - desiredClientVersionMinor ))
+
+    if [[ $minorVersionDifference -gt 1 ]] || [[ $minorVersionDifference -lt -1 ]] || [[ $currentClientVersionMajor -ne $desiredClientVersionMajor ]]; then
```

Comment:
info from @pbochynski : print only a warning that kubectl version should be grater than ...in this case grater than 1.9.0